pubspec.yaml

```diff
@@ -20,19 +20,24 @@ environment:
   sdk: ">=1.14.0 <2.0.0"
 
 dependencies:
+  dart2_constant: ^1.0.1
   fluri: ^1.2.2
   http_parser: '>=2.2.0 <4.0.0'
   meta: ^1.1.0
   mime: ^0.9.3
   sockjs_client:
     git:
-      url: https://github.com/Workiva/sockjs-dart-client.git
-      ref: 0.3.2
+      url: git://github.com/Workiva/sockjs-dart-client.git
```

Comment:
FYI I had to edit this to `git@github.com:Workiva/sockjs-dart-client.git`
otherwise I got a
```
Resolving dependencies...
Git error. Command: git fetch
fatal: not a git repository (or any of the parent directories): .git
```

Reply:
Hmm, that’s odd.. I had Issues with the `git@github.com:` format on Dart 2 and had to switch to this approach to get it working. I’ll try it out again locally on Dart 1 and 2 just to make sure, maybe I had something weird going on

Reply:
I think it's mostly due to my SSO / SSH key setup .. since it passed on CI I'm not worried.